website/docs/security/authentication.md

```diff
@@ -6,8 +6,21 @@ description: Learn to use authentication mechanisms
 
 # Authentication
 
-FerretDB does not store authentication information (usernames and passwords) itself but uses the backend's authentication mechanisms.
-The default username and password can be specified in FerretDB's connection string,
+The authentication is used to verify the identity of the client connecting to FerretDB using username and password.
+It blocks the access if the client does not have the valid credentials.
+
+FerretDB provides authentication by relying on the [backend's authentication](#postgresql-backend-authentication) mechanisms by default.
+However, there is an [experimental authentication](#experimental-authentication) feature that allows managing and authenticating users within FerretDB.
+
+Having user management within FerretDB allows creations and deletions of users via command and hides the necessity to have the knowledge of the specific backend to manage users.
```

Comment:
"via command" looks a bit strange. That's not a single command. I would say "via normal MongoDB commands" or something